eng/pipelines/templates/jobs/build-validate-pom.yml

```diff
@@ -41,11 +41,16 @@ jobs:
 
       - template: /eng/common/pipelines/templates/steps/daily-dev-build-variable.yml
 
-      - script: |
-          pip install setuptools wheel
-          pip install doc-warden==$(DocWardenVersion)
-          ward scan -d $(Build.SourcesDirectory) -c $(Build.SourcesDirectory)/eng/.docsettings.yml
+      - task: PowerShell@2
         displayName: 'Verify Readmes'
+        inputs:
+          pwsh: true
+          workingDirectory: $(Agent.BuildDirectory)
+          filePath: eng/common/scripts/Verify-Readme.ps1
+          arguments: >
+            -DocWardenVersion $(DocWardenVersion)
```

Comment:
As part of putting this into common I'd actually like use to remove the DocWardenVersion from all the languages as well. We should just include it as part of the Verify-Readme.ps1 and let it declare the version instead of repeating that in all repos. 

FYI @chidozieononiwu @sima-zhu